src/tool_getparam.c

```diff
@@ -1680,16 +1681,16 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
         break;
 
       case 'u': /* TLS username for proxy */
-        if(curlinfo->features & CURL_VERSION_TLSAUTH_SRP)
-          GetStr(&config->proxy_tls_username, nextarg);
-        else
+        GetStr(&config->proxy_tls_username, nextarg);
+        cleanarg(nextarg);
```

Comment:
FWIW I'm in favor of scrubbing TLS username as well. It's cheap enough that we can do that to possibly make life just a tiny bit harder for anyone with malicious intents.

Reply:
OK, agreed.